### DISCUSSION TITLE
Can we expand this repo by including Gen AI classes as well?

### DISCUSSION THREAD

**Discussion by davidatuken** [apachecn/MachineLearning#651] [Q&A] [answered]
Keep here a record of Gen AI classes that may be translated to Chinese and expand the reach of this repo

**Comment by zyuegege** [comment]
这是来自张越的自动回复邮件。您好，你的邮件已经收到，我会尽快给您回复。

**Comment by david-wei-01001** [comment, marked as answer]
Some potential resources to consider includes:
https://github.com/microsoft/generative-ai-for-beginners
https://github.com/AzureCosmosDB/Azure-OpenAI-Node.js-Developer-Guide?tab=readme-ov-file
https://www.youtube.com/watch?v=i0bjpRHHyLA
